server/liveblog/tests/environment.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from apps.io.tests import setup_providers
+from superdesk import tests
+from superdesk.tests import setup_auth_user
+from superdesk.tests.environment import setup_search_provider
+from superdesk.vocabularies.command import VocabulariesPopulateCommand
+from liveblog.tests import setup_auth_consumer, test_consumer
+
+
+def setup_before_scenario(context, scenario, config, app_factory):
```

Comment:
I think that is better to have our environment.py and remove dependency with Superdesk tests. If you want to improve the Superdesk project with our changes instead, we can discuss about that, but I don't think that you need Syndication testing environment and scenarios.